plugins/command-not-found/command-not-found.plugin.zsh

```diff
@@ -26,10 +26,8 @@ fi
 
 # OSX command-not-found support
 # https://github.com/Homebrew/homebrew-command-not-found
-if type brew &> /dev/null; then
-  if brew command command-not-found-init > /dev/null 2>&1; then
-    eval "$(brew command-not-found-init)";
-  fi
+if [[ -s '/usr/local/Homebrew/Library/Taps/homebrew/homebrew-command-not-found/handler.sh' ]]; then
```

Comment:
This also speeds up this plugin for me.
I wonder if a variable might mitigate some of the loss of robustness.
```suggestion
if [[ -v HOMEBREW_COMMAND_NOT_FOUND_REPO ]]; then
HOMEBREW_COMMAND_NOT_FOUND_REPO='/usr/local/Homebrew/Library/Taps/homebrew/homebrew-command-not-found/`
fi

if [[ -s "$HOMEBREW_COMMAND_NOT_FOUND_REPO/handler.sh" ]]; then
```